test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -50,7 +52,16 @@ auto dpcpp_policy()
 }
 #endif //TEST_DPCPP_BACKEND_PRESENT
 
-auto host_policies() { return std::true_type{};}
+struct AllPolicies{};
+struct DeviceAndParPolicies{};
+struct DevicePolicy{};
+
+template <typename T>
+struct is_host_policy_selector: std::bool_constant<std::is_same_v<T, AllPolicies> ||
```

Comment:
Probably some construction with using `std::disjunction` like usen in the implementation of `__has_known_identity` may be more clear....

Reply:
Also I think the usage of `std::bool_constant` really isn't required here at all.